OrbitGl/TimeGraph.cpp

```diff
@@ -626,12 +626,10 @@ void TimeGraph::Draw(GlCanvas* canvas, PickingMode picking_mode) {
 
 namespace {
 
-std::string GetLabelBetweenIterators(const TextBox* box_a,
-                                     const TextBox* box_b) {
-  std::string function_from =
-      Capture::GAddressToFunctionName[box_a->GetTimerInfo().function_address()];
-  std::string function_to =
-      Capture::GAddressToFunctionName[box_b->GetTimerInfo().function_address()];
+std::string GetLabelBetweenIterators(const FunctionInfo& function_a,
```

Comment:
nit: this should probably be [[nodiscard]]

Reply:
Yes, thanks for spotting this!